staging/src/k8s.io/api/extensions/v1beta1/types.go

```diff
@@ -210,7 +210,7 @@ type RollingUpdateDeployment struct {
 	// Value can be an absolute number (ex: 5) or a percentage of desired pods (ex: 10%).
 	// Absolute number is calculated from percentage by rounding down.
 	// This can not be 0 if MaxSurge is 0.
-	// By default, a fixed value of 1 is used.
```

Comment:
revert this change. the default in extensions/v1beta1 is 1:
https://github.com/kubernetes/kubernetes/blob/0c0408c790c64d9105a85ae702c73d0714bdb850/pkg/apis/extensions/v1beta1/defaults.go#L104-L108